docs/pages/model_hub/speaker_diarization.md

```diff
@@ -0,0 +1,34 @@
+# Speaker Diarization (SD) Models
+
+[SpeakerDiarizationDeployment](./../../reference/deployments.md#aana.deployments.SpeakerDiarizationDeployment) allows you to diarize the audio for speakers audio with pyannote models. The deployment is based on the [pyannote.audio](https://github.com/pyannote/pyannote-audio) library.
+
+[SpeakerDiarizationConfig](./../../reference/deployments.md#aana.deployments.SpeakerDiarizationConfig) is used to configure the Speaker Diarization deployment.
+
+::: aana.deployments.SpeakerDiarizationConfig
+    options:
+        show_bases: false
+        heading_level: 4
+        show_docstring_description: false
+        docstring_section_style: list
+
+### Example Configurations
+
+As an example, let's see how to configure the Speaker Diarization deployment for the [Speaker Diarization-3.1 model](https://huggingface.co/pyannote/speaker-diarization-3.1).
+
+Note that pyannote speaker diarization models are gated. Get access to the model from (https://huggingface.co/pyannote/speaker-diarization-3.1) and use HuggingFace access token via setting environment variable in the SDK (`HF_TOKEN` variable in `.env`) before using the model.
```

Comment:
I would suggest to add a bit more information here. Something like:

```md

The PyAnnote speaker diarization models are gated, requiring special access. To use these models:

1. **Request Access**:  
    Visit the [PyAnnote Speaker Diarization 3.1 model page](https://huggingface.co/pyannote/speaker-diarization-3.1) on Hugging Face. Log in, fil out the form, and request access.

2. **Approval**:  
    - If automatic, access is granted immediately.
    - If manual, wait for the model authors to approve your request.

3. **Set Up the SDK**:  
    After approval, add your Hugging Face access token to your `.env` file by setting the `HF_TOKEN` variable:

    ```plaintext
    HF_TOKEN=your_huggingface_access_token
    ```

    To get your Hugging Face access token, visit the [Hugging Face Settings - Tokens](https://huggingface.co/settings/tokens).
```

I still think it might be good to have a page in the docs regarding gated HF models so we don't need to repeat it.

Reply:
Added the above content as a section in the speaker_diarization deployment. Feel free to open an issue if the idea is to go with a specific page.